BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -249,13 +255,14 @@ extension AttributedLabel {
 
             if !isMeasuring {
```

Comment:
I nested these all under `!isMeasuring`, since none of them matter for measuring AFAICT.